docs/deploy/restate_cloud.md

```diff
@@ -7,62 +7,65 @@ import Admonition from '@theme/Admonition';
 
 # Restate Cloud
```

Comment:
@jackkleeman is the cloud documentation page up to date with the actual cloud state? The changes in this PR are only of cosmetic nature.

Reply:
i dont really think we should have a cloud docs page right now, this is about the old cloud POC

Reply:
@jackkleeman should we remove it?

Reply:
id say yes just remove the page and we will write something new for *actual* cloud towards launch

Reply:
Ok, will remove it then and create a follow-up ticket to add the new documentation.

Reply:
for posterity: https://github.com/restatedev/documentation/issues/357